go-apps/meep-ams/server/ams.go

```diff
@@ -2001,3 +2022,22 @@ func populateAppInstanceIds(key string, fields map[string]string, response inter
 	//response = &resp
 	return nil
 }
+
+func validateAppInstanceId(appInstanceId string) (error, int, string) {
```

Comment:
You already have a list of App Instance IDs that you obtained from periodic polling the Sandbox Controller applications endpoint. Simply check that the app instance exists in the map instead of looking in redis database.

Reply:
Fixed.